server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -1207,9 +1207,9 @@ def _enrich_kafka_topic_stats(
 
     async def _get_function_summary_applications(
         self,
+        start: datetime,
+        end: datetime,
```

Comment:
please also update `function_summaries` (L899-L900) and `function_summary` (L971-L972) and make these params mandatory